relay-dynamic-config/src/defaults.rs

```diff
@@ -15,6 +15,9 @@ const MOBILE_OPS: &[&str] = &["app.*", "ui.load*"];
 /// A list of patterns found in MongoDB queries
 const MONGODB_QUERIES: &[&str] = &["*\"$*", "{*", "*({*", "*[{*"];
 
+/// A list of patterns for resource span ops we'd like to ingest.
+const RESOURCE_SPAN_OPS: &[&str] = &["resource.script", "resource.css"];
```

Comment:
@phacops why are we restricting to these two? What about `resource.link`?

Reply:
This is from a conversation with @AbhiPrasad on Slack. I think it's fine to open up to `resource.link` as well.